.github/workflows/continuous-integration.yml

```diff
@@ -55,113 +55,114 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         promttoolkit: [3.*, 2.*]
         include:
           - promttoolkit: 3.0.29
             os: ubuntu-latest
-            python-version: '3.10'
           - promttoolkit: 3.0.19
             os: ubuntu-latest
-            python-version: '3.10'
 
     steps:
-    - name: Checkout git repository 🕝
-      uses: actions/checkout@v3
-
-    - name: Set up Python ${{ matrix.python-version }} 🐍
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ matrix.python-version }}
-
-    - name: Install poetry 🦄
-      uses: Gr1N/setup-poetry@v7
-
-    - name: Load Poetry Cached Libraries ⬇
-      uses: actions/cache@v3
-      with:
-        path: ~/.cache/pypoetry
-        key: ${{ runner.os }}-poetry-${{ hashFiles('**/poetry.lock') }}
-        restore-keys: ${{ runner.os }}-poetry-
-
-    - name: Install dependencies 🖥
-      run: |
-        poetry install --no-interaction
-        poetry run pip install prompt_toolkit==${{ matrix.promttoolkit }}
-
-    - name: Test Code 🔍
-      run: make test
-
-    - name: Send Coverage Report 📊
-      env:
-        COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        COVERALLS_SERVICE_NAME: github
-        COVERALLS_PARALLEL: true
-      run: poetry run coveralls
-
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }} 🐍
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install poetry 🦄
+        uses: Gr1N/setup-poetry@v8
+
+      - name: Load Poetry Cached Libraries ⬇
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pypoetry
+          key: ${{ runner.os }}-poetry-${{ hashFiles('**/poetry.lock') }}
+          restore-keys: ${{ runner.os }}-poetry-
+
+      - name: Install dependencies 🖥
+        run: |
+          poetry install --no-interaction
+          poetry run pip install prompt_toolkit==${{ matrix.promttoolkit }}
+
+      - name: Test Code 🔍
+        run: make test
+
+      - name: Send Coverage Report 📊
+        shell: bash -l {0}
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COVERALLS_SERVICE_NAME: github
+          COVERALLS_SERVICE_JOB_ID: ${{ github.run_id }}
+          COVERALLS_SERVICE_NUMBER: "${{ github.workflow }}-${{ github.run_number }}"
+          COVERALLS_PARALLEL: true
+        run: poetry run coveralls
```

Comment:
Any idea why coveralls isn't posting a status on the PR anymore? Is it related to these changes? See this PR for an example of what used to happen https://github.com/tmbo/questionary/pull/244.

Also, source code is not visible in coveralls, but perhaps this was always the case e.g., https://coveralls.io/builds/60155724/source?filename=questionary%2Fprompts%2Fpath.py#L69. Is this related?

Reply:
I don't have any experience with coveralls, this was actually a misguided change stemming from the discussion here:

https://github.com/TheKevJames/coveralls-python/issues/252

In reality, I just obviously didn't have permissions to the endpoint 😄 . We can remove these changes if you'd like.

Reply:
Try removing and see if the status update comes back. Otherwise, I'll look at why the status has disappeared in a separate PR!